prefect_hex/credentials.py

```diff
@@ -30,6 +30,7 @@ class HexCredentials(Block):
         default="app.hex.tech", description="Domain to make API requests against."
     )
     token: SecretStr = Field(default=..., description="Token used for authentication.")
+    _code_example = ""
```

Comment:
Curious why we want to override example with nothing

Reply:
Good eye! I was experimenting with something an accidentally left this in.